lightkurve/targetpixelfile.py

```diff
@@ -1336,7 +1336,9 @@ def __init__(self, path, quality_bitmask='default', **kwargs):
         # Early TESS releases had cadences with time=NaN (i.e. missing data)
         # which were not flagged by a QUALITY flag yet; the line below prevents
         # these cadences from being used. They would break most methods!
-        self.quality_mask &= np.isfinite(self.hdu[1].data['TIME'])
+        #self.quality_mask &= np.isfinite(self.hdu[1].data['TIME'])  # kjm2019JAN21: original
+        if (quality_bitmask > 0):  # kjm2019: new
```

Comment:
Some of the unit tests fail here ([link](https://ci.appveyor.com/project/mirca/lightkurve/builds/21781622/job/n9vnkif81k7xs64l#L591)) because `quality_bitmask` can be either a string ('none', 'default', 'hard', or 'hardest') or a number.

I recommend changing 
```
if (quality_bitmask > 0):
```
into
```
if (quality_bitmask != 0) and (quality_bitmask != 'none'):
```